model/registry/cpu.yaml

```diff
@@ -0,0 +1,38 @@
+groups:
+  - id: registry.cpu
+    prefix: cpu
+    type: attribute_group
+    brief: >
+      A cpu instance
```

Comment:
is there a more specific description we can provide? Check out how it's rendered in registry markdown.

Reply:
I can try, but not sure how much expressive we can be here.